src/components/scroll-to-top.vue

```diff
@@ -1,19 +1,20 @@
 <template>
-  <BaseScrollToTop class="x-button--round" scrollId="main-scroll" :thresholdPx="100">
+  <ScrollToTop class="x-button--round" scrollId="main-scroll" :thresholdPx="100">
```

Comment:
Do we really need this component just to use a ScrollToTop with an icon?

Reply:
yes